rswag-specs/lib/rswag/specs/swagger_formatter.rb

```diff
@@ -126,19 +126,31 @@ def upgrade_response_produces!(swagger_doc, metadata)
         target_node = metadata[:response]
         upgrade_content!(mime_list, target_node)
         metadata[:response].delete(:schema)
+        metadata[:response].delete(:examples)
       end
 
       def upgrade_content!(mime_list, target_node)
         target_node.merge!(content: {})
         schema = target_node[:schema]
-        return if mime_list.empty? || schema.nil?
+        examples = upgrade_examples(target_node[:examples])
+        return if mime_list.empty? || (schema.nil? && examples.nil?)
 
+        content = { schema: schema, examples: examples }.select { |_, v| v.present? }
         mime_list.each do |mime_type|
           # TODO upgrade to have content-type specific schema
-          target_node[:content][mime_type] = { schema: schema }
+          target_node[:content][mime_type] = content       
         end
       end
 
+      def upgrade_examples(examples)
+        return unless examples.present?
+        examples.map do |k, v| 
```

Comment:
so the fix is to convert 
```
{ 
  	k: v,
	...
}
```

to 
```
{
	k: {
		value: v
	},
	...
}
```
?

Reply:
yeah, thats how the example blocks look in the spec. https://swagger.io/docs/specification/adding-examples/
If you checkout the section about multiple examples for a response, it shows the value key there.